.gitignore

```diff
@@ -1,4 +1,6 @@
 site/
 .DS_Store
 src/.DS_Store
-src/04-modality-specific-files/.DS_Store
\ No newline at end of file
+src/04-modality-specific-files/.DS_Store
```

Comment:
could you change the several `.DS_Store` lines into a single line that covers `.DS_Store` everywhere, please?

Reply:
Should be good now.